.github/workflows/publish.yaml

```diff
@@ -61,20 +64,29 @@ jobs:
       # Standard Package by default tests what's in the Istio Package
       - name: Test amd64 Bundle
         run: |
-          uds run deploy-standard-bundle --no-progress
+          uds run -f tasks/deploy.yaml k3d-standard-bundle --no-progress
```

Comment:
Will want to switch this back to `uds run deploy-standard-bundle --no-progress` - otherwise the k3d image doesn't get populated from the default var in `tasks.yaml`.

Reply:
@mjnagel - updated now